lib/mirage_nat.ml

```diff
@@ -31,19 +35,14 @@ module type S = sig
   type t
   type config
 
-  type insert_result =
-    | Ok
-    | Overlap
-    | Unparseable
-
   val empty : config -> t Lwt.t
 
   (** given a lookup table, rewrite direction, and an ip-level frame,
     * perform any translation indicated by presence in the table
     * on the Cstruct.t .  If the packet should be forwarded, return Some packet,
     * else return None.
     * This function is zero-copy and mutates values in the given Cstruct.  *)
```

Comment:
Update this docstring to reflect the change in signature?  Also the function is no longer zero-copy AFAIU.